src/ray/core_worker/core_worker.cc

```diff
@@ -8,9 +8,13 @@
 #include "ray/core_worker/context.h"
 #include "ray/core_worker/transport/direct_actor_transport.h"
 #include "ray/core_worker/transport/raylet_transport.h"
+#include "ray/util/util.h"
 
 namespace {
 
+// Duration between internal book-keeping heartbeats.
+const int kInternalHeartbeatMillis = 1000;
```

Comment:
Should add this to `RayConfig`?

Reply:
I was thinking this would never need to change. Perhaps we can move it if it ever does need to be updated?